File: Data Structures & Algorithms/is-anagram/submission-1.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        if (t.length()!=s.length()){return false;}
        int arr[26] = {0};
        for(int i; i<s.length(); i++){
            arr[s[i]-'a']++;
            arr[t[i]-'a']--;
        }
        for (int v : arr){
            if (v != 0){
                return false;
            }
        }
        return true;
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        hashmap = defaultdict(int)
        for n in nums:
            hashmap[n] += 1
        arr = []
        for num, amt in hashmap.items():
            arr.append([amt, num])
        arr.sort()
        res = []
        for i in range(k):
            res.append(arr.pop()[1])
        return res